content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md

```diff
@@ -316,6 +316,10 @@ To build this link, you'll need your OAuth Apps `client_id` that you received fr
 
 {% endtip %}
 
+## Limitations
+
+There is no support for PKCE ([RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636)), see [feature request](https://github.com/orgs/community/discussions/15752).
```

Comment:
This is more appropriate around line 57, where a developer would be looking for the supported parameters.

Reply:
Perhaps "The PKCE (Proof Key for Code Exchange ) parameters `code_challenge` and `code_challenge_method` are not supported at this time" and leave it at that. Linking to the feedback discussion isn't standard for our documentation.

Reply:
Done.

Reply:
Thank you!